.bazelrc

```diff
@@ -142,6 +142,10 @@ build --config=short_logs
 
 build --config=v2
 
+# Disable AWS/HDFS support by default
+build --define=no_aws_support=true
+build --define=no_hdfs_support=true
+
```

Comment:
@yongtang I don't see these options being used anywhere. Is this for future reference?